include/circt/Dialect/FIRRTL/AnnotationDetails.h

```diff
@@ -91,6 +91,10 @@ constexpr const char *extractGrandCentralClass =
     "sifive.enterprise.grandcentral.ExtractGrandCentralAnnotation";
 constexpr const char *signalDriverAnnoClass =
     "sifive.enterprise.grandcentral.SignalDriverAnnotation";
+constexpr const char *signalDriverTargetAnnoClass =
+    "sifive.enterprise.grandcentral.SignalDriverAnnotation.target";
```

Comment:
```suggestion
    "sifive.enterprise.grandcentral.SignalDriverAnnotation.target"; // not in SFC
```
(unsure re:wrapping)